### ISSUE TITLE
Refund "in progress" status shows translation key

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2284]
### Expected behavior
Should show "Refunding" or similar message in my language

### Actual Behavior
![reaction](https://cloud.githubusercontent.com/assets/72819/26131640/ba28b5c6-3acd-11e7-84cf-ccc04a1c9373.png)

### Steps to Reproduce the Behavior
1. Process an order using a method that supports refunds (Stripe/Paypal)
1. Approve and Capture the order
1. Enter a refund.
1. During the "in progress" state, observe the translation key

### Versions
```
Node: 7.10.0
NPM: 4.2.0
Meteor Node: 4.8.2
Meteor NPM: 4.5.0
Reaction CLI: 0.8.2
Reaction: 1.2.0
Reaction branch: release-1.2.0
Docker: 17.03.1-ce
```

**Comment by aaronjudd** [comment]
@kieckhafer was this resolved in #2302 ? @michael-alade should be as simple as adding the key?

**Comment by kieckhafer** [comment]
Yes, this is fixed in #2302. Closing.
